pkg/chrootarchive/archive_unix.go

```diff
@@ -32,6 +32,7 @@ func untar() {
 	flag.Parse()
 
 	var options *archive.TarOptions
+	options.InUserNS = true
```

Comment:
This is the cause of the test panic as `options` is still **nil** at this point.

However, I assume this is actually being passed in via the JSON "options" object and shouldn't be set here anyway, so maybe a stray leftover from testing?